### DISCUSSION TITLE
How to draw `bottomAxis` labels below `startAxis`?

### DISCUSSION THREAD

**Discussion by mak1nt0sh** [patrykandpatrick/vico#735] [Questions]
### Question

Is it possible to draw labels below vertical axis like so? This would help the line start at the beginning of axis and not leave huge padding like `addExtremeLabelPadding` does.

<img width="466" alt="Screenshot 2024-05-28 at 20 15 24" src="https://github.com/patrykandpatrick/vico/assets/33786477/d4cdbf37-3d27-4798-a263-cb6907d2ff4b">

### Vico version(s)

2.0.0-alpha.20

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @mak1nt0sh. It's not currently possible, but it's a planned addition. In the meantime, you can achieve this by creating a `HorizontalAxis` subclass.
